src/python/pants/backend/python/util_rules/pex_test.py

```diff
@@ -438,9 +454,7 @@ def test_pex_execution(rule_runner: RuleRunner) -> None:
     assert result.stdout == b"from main\n"
 
 
-# TODO(John Sirois): Add VenvPex to the pex_type parameter list once Pants is upgraded to Pex with
-#  a fix for: https://github.com/pantsbuild/pex/issues/1239
-@pytest.mark.parametrize("pex_type", [Pex])
+@pytest.mark.parametrize("pex_type", [Pex, VenvPex])
```

Comment:
The VenvPex parametrization fails trying to extract PEX-INFO with PEX_TOOLS before the script fix.